client/platform/web-girder/api/admin.service.ts

```diff
@@ -0,0 +1,51 @@
+// Admin endpoints used for review and managment
+import girderRest from 'platform/web-girder/plugins/girder';
+import type { GirderJob, GirderModel } from '@girder/components/src';
+
+export interface JobTypeStatus {
+  status: number[];
+  types: string[];
+}
+
+function getRecentDatasets(limit: number, offset: number) {
+  return girderRest.get(`dive_dataset?limit=${limit}&sort=created&sortdir=-1&offset=${offset}&published=false&shared=false`);
+}
```

Comment:
Probably not for this PR but should we also be able to filter datasets based on media type?

Reply:
This could be added as an optional parameter in the future.  It can be logged as a new feature request if we want to do it.